src/ir/block_builder.h

```diff
@@ -104,6 +109,22 @@ class BlockBuilder {
     return std::move(block_);
   }
 
+  // A way to get non-owning access to the block (for reference by operations)
+  // while the block is still under construction.
+  Block* get_block_ptr() {
```

Comment:
GetBlockPtr()

Reply:
Done!
Though I find Java style method names less readable, thanks for the consistency notes.
Should we change other methods too or is this context dependent?